### PR TITLE
API Chore Housekeeping

### PR DESCRIPTION
**Pull request by maxarvid**
Link to PT: https://www.pivotaltracker.com/story/show/180157527

### User Story
```
As an API
In order to keep up with my frontends
I would like to: 
- Use a FactoryBot that reflects the latest changes
- Add a top_story attribute to the articles table
- Return the article authors as a sentence instead of as an array
```
### Tasks
- Add spec to check that `authors` can be returned as sentence
- Refactor serializers to return `authors_as_sentence`
- Refactor serializer specs accordingly
- Add `top_story` attribution migration to the database
- Refactor the FactoryBot to not longer create `category_name` or `category_id`

### CODE REVIEW

File: app/serializers/articles/index_serializer.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Articles::IndexSerializer < ActiveModel::Serializer
   include Rails.application.routes.url_helpers
-  attributes :id, :title, :lede, :updated_at, :published, :image
+  attributes :id, :title, :lede, :updated_at, :published, :image, :authors_as_sentence
```
Comment (Fraseer) [inline app/serializers/articles/index_serializer.rb]: pls can you add  top story here

File: spec/serializers/show_serializer_spec.rb [hunk 2]
```diff
@@ -32,7 +32,8 @@
           'name' => an_instance_of(String)
         },
         'authors' => an_instance_of(Array),
-        'image' => an_instance_of(String)
+        'image' => an_instance_of(String),
+        'authors_as_sentence' => an_instance_of(String)
```
Comment (Fraseer) [inline spec/serializers/show_serializer_spec.rb:36]: and this here `'top_story' => an_instance_of(FalseClass),`

**Review by Fraseer — CHANGES_REQUESTED** [review]
Looks good, just a couple additions

File: app/serializers/articles/index_serializer.rb (hunk 1 above)
Comment (maxarvid) [reply app/serializers/articles/index_serializer.rb]: Added!

File: spec/serializers/show_serializer_spec.rb (hunk 2 above)
Comment (maxarvid) [reply spec/serializers/show_serializer_spec.rb:36]: Added!

**Review by Fraseer — APPROVED** [review]
Cool good changes

**Review by AugustEnhager — APPROVED** [review]

**Pull request merged**